### PR TITLE
adding SAN support  #61

### PR DESCRIPTION
**Pull request by Neetuj**
As per yesterdays discussion #27 is being broken into to tasks
1. adding SAN
2. figuring out the ID ( if we want to bese64 encode it or not) 

this commit takes care of first part

### CODE REVIEW

**Comment by beittenc** [comment]
:+1:

File: jumpgate/compute/drivers/sl/servers.py [hunk 1]
```diff
@@ -201,6 +201,7 @@ def on_post(self, req, resp, tenant_id):
             'domain': CONF['default_domain'] or 'jumpgate.com',
             'cpus': flavor['cpus'],
             'memory': flavor['ram'],
+            'local_disk': True if flavor['disk-type'] is "local" else False,
```
Comment (sudorandom) [inline jumpgate/compute/drivers/sl/servers.py]: 'local' does not equal 'Local' so local_disk will never be True as the code stands now

Comment (Neetuj) [inline jumpgate/compute/drivers/sl/servers.py]: good catch :) .. will correct it

**Comment by sudorandom** [comment]
+1

I tested this by:
- listing Flavors with the nova CLI
- ordering both SAN and local disk flavors with the nova CLI
- Show quotas with the nova CLI

```
$ nova flavor-list
+----+-------------------------------+-----------+------+-----------+------+-------+-------------+-----------+
| ID | Name                          | Memory_MB | Disk | Ephemeral | Swap | VCPUs | RXTX_Factor | Is_Public |
+----+-------------------------------+-----------+------+-----------+------+-------+-------------+-----------+
| 1  | 1 vCPU, 1GB ram, 25GB, local  | 1024      | 25   | 0         |      | 1     | 1           | True      |
| 2  | 1 vCPU, 1GB ram, 100GB, local | 1024      | 100  | 0         |      | 1     | 1           | True      |
| 3  | 2 vCPU, 2GB ram, 100GB, local | 2048      | 100  | 0         |      | 2     | 1           | True      |
| 4  | 4 vCPU, 4GB ram, 100GB, local | 4096      | 100  | 0         |      | 4     | 1           | True      |
| 5  | 8 vCPU, 8GB ram, 100GB, local | 8192      | 100  | 0         |      | 8     | 1           | True      |
| 11 | 1 vCPU, 1GB ram, 25GB, SAN    | 1024      | 25   | 0         |      | 1     | 1           | True      |
| 12 | 1 vCPU, 1GB ram, 100GB, SAN   | 1024      | 100  | 0         |      | 1     | 1           | True      |
| 13 | 2 vCPU, 2GB ram, 100GB, SAN   | 2048      | 100  | 0         |      | 2     | 1           | True      |
| 14 | 4 vCPU, 4GB ram, 100GB, SAN   | 4096      | 100  | 0         |      | 4     | 1           | True      |
| 15 | 8 vCPU, 8GB ram, 100GB, SAN   | 8192      | 100  | 0         |      | 8     | 1           | True      |
+----+-------------------------------+-----------+------+-----------+------+-------+-------------+-----------+
```

Order local disk:

```
nova boot --flavor=1 --image=GUID
```

Order SAN:

```
nova boot --flavor=11 --image=GUID
```

Show quotas

```
$ nova quota-show
+-----------------------------+--------+
| Quota                       | Limit  |
+-----------------------------+--------+
| instances                   | 10     |
| cores                       | 200    |
| ram                         | 512000 |
| floating_ips                | 100    |
| metadata_items              | 128    |
| injected_files              | 5      |
| injected_file_content_bytes | 10240  |
| injected_file_path_bytes    | 255    |
| key_pairs                   | 100    |
| security_groups             | 10     |
| security_group_rules        | 20     |
+-----------------------------+--------+
```

**Pull request merged**